src/errors/meilisearch-api-error.ts

```diff
@@ -1,22 +1,19 @@
-import { MSApiError, MeiliSearchApiErrorResponse } from '../types'
```

Comment:
No more `MeiliearchApiError`?

Reply:
`MsApirError` and `MeiliSearchApiErrorResponse` have been replaced with `MeiliSearchErrorInterface`

> const MeiliSearchApiError = class extends Error 

Naming has changed for the types but the class still exists. I'm not sure what you are asking :S

Reply:
Discussed with @bidoubiwa, it's solved, a miss understanding on my side.